src/main/java/org/wildfly/security/auth/util/GSSCredentialSecurityFactory.java

```diff
@@ -281,6 +283,20 @@ public Builder setOptions(final Map<String, Object> options) {
         }
 
         /**
+         * Set amount of seconds before new try to obtain {@link GSSCredential} should be done if it has failed last time.
+         * Allows to prevent long waiting to unavailable KDC on every authentication on IBM java.
```

Comment:
Not a big deal but we can probably avoid mentioning IBM here, I suspect if obtain Kerberos Ticket was set to true this would be more apparent on Oracle and OpenJDK.